slang-antlr/src/main/antlr4/com/sonarsource/slang/parser/SLang.g4

```diff
@@ -127,6 +129,21 @@ expressionName
   :  identifier
   ;
 
+conditional
+  :  IF LPAREN statementOrExpression RPAREN controlBlock (ELSE controlBlock)?
+  |  MATCH LPAREN statementOrExpression RPAREN LCURLY matchCase* RCURLY
```

Comment:
I think we should have 2 separate grammar rules for "if" and "match": it would make it slightly easier to convert the AST.